scalatest/src/main/scala/org/scalatest/tools/DiscoverySuite.scala

```diff
@@ -66,7 +67,7 @@ private[scalatest] object DiscoverySuite {
         clazz.newInstance.asInstanceOf[Suite]
       else {
         val suiteClazz = wrapWithAnnotation.value
-        val constructorList = suiteClazz.getDeclaredConstructors()
+        val constructorList : Array[Constructor[_]] = suiteClazz.getDeclaredConstructors().map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.